config/config-mp/src/main/java/io/helidon/config/mp/MpMetaConfigUtils.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (c) 2022 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.config.mp;
+
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.function.BiFunction;
+import java.util.function.Function;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import io.helidon.config.ConfigException;
+import io.helidon.config.ConfigValue;
+
+import org.eclipse.microprofile.config.spi.ConfigSource;
+
+/**
+ * Utilities for Helidon MicroProfile Meta-Config implementation.
+ */
+public class MpMetaConfigUtils {
```

Comment:
Does this class need to be public?

Reply:
This is being used by HoconMpMetaConfigProvider and YamlMetaConfigProvider which are both on their own module, i.e. hocon-mp and yaml-mp respectively. So I think because of that, it needs to be public, right?